modules/configuring-gpu-acceleration-on-your-cluster.adoc

```diff
@@ -0,0 +1,62 @@
+:_module-type: PROCEDURE
+
+[id="configuring-gpu-acceleration-openshift-cluster_{context}"]
+= Configuring GPU acceleration on an OpenShift cluster
+
+You must configure GPU acceleration on your {openshift-platform} cluster to support efficient large language model (LLM) inference, such as when using vLLM. vLLM is optimized for NVIDIA GPUs and does not perform efficiently on CPU-only nodes.
+
+.Prerequisites
+
+* You have cluster administrator privileges.
+* You have access to the OpenShift web console and the OpenShift CLI (`oc`).
```

Comment:
make consistent with lines 9-14 in the Installing the Llama Stack operator module.